CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * Fix [#214](https://g1cp.org/issues/214): Graham now correctly sits at the campfire in the evening.
 * Fix [#215](https://g1cp.org/issues/215): Guy now correctly sits at the arena in the evening.
 * Fix [#216](https://g1cp.org/issues/216): A digger now correctly repairs his hut during daytime.
+* Fix [#217](https://g1cp.org/issues/217): A mercenary now correctly guards the palisade at the Free Mine in the afternoon.
```

Comment:
As the line above (#216) the phrasing "**A** .... now correctly does xyz." may be a bit misleading and could be understood as if we *added* a new NPC to do that task, for which there was no NPC beforehand. @AmProsius what do you think about phrasing it more like "The gate guard ... " a few lines above (#182):

"**One of the** mercenaries ... " and "**One of the** diggers **in the Old Camp** ... "

or alternatively

"The mercenary guarding the palisade at the Free Mine now correctly follows his daily routine in the afternoon."